operational_graphics/toolboxes/scripts/CanvasAreaGRG.py

```diff
@@ -36,6 +36,7 @@
 outputFeatureClass = arcpy.GetParameterAsText(6)
 labelStartPos = arcpy.GetParameterAsText(7)
 tempOutput = "in_memory" + "\\" + "tempFishnetGrid"
+sysPath = sys.path[0]
```

Comment:
We also use: os.path.dirname(__file__) though I am not sure how/if these behave differently.

You may want to just do the `os.path.dirname()` here